src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1559,7 +1528,7 @@ public void recallRequestNoticeIsSentWhenPolicyDefinesRecallRequestNoticeConfigu
       "ItemCN",
       "ItemPrefix",
       "ItemSuffix",
-      Arrays.asList("CopyNumbers", "CopyDetails"));
+      "CopyNumbers");
```

Comment:
There can only be one copy number now. Maybe this string should be changed to reflect that?